code/modules/research/designs/medical_designs.dm

```diff
@@ -1135,34 +1147,64 @@
 	surgery = /datum/surgery/advanced/brainwashing
 	research_icon_state = "surgery_head"
 
+/datum/design/surgery/brainwashing/mechanic
+	name = "Reprogramming"
+	desc = "Malware which directly implants a directive into the robotic patient's operating system, making it their absolute priority. It can be cleared using a mindshield implant."
```

Comment:
Same for these I guess but maybe I am being a little too anal-retentive about the actual mechanics of what is happening. I can at least see "installing some software into the head which effects the brain" as effective I guess.

Reply:
It's _mildly_ semantic with brainwashing, and yeah, my interpretation is basically 'the brains connected to a bunch of wires and machines so in a way you have to 'program' it as an alternative to just cutting up the brain itself'.